dev-util/kak_lsp/kak_lsp-12.1.0.recipe

```diff
@@ -0,0 +1,449 @@
+SUMMARY="Kakoune Language Server Protocol Client"
+DESCRIPTION="\
+Kak-lsp is a Language Server Protocol client for Kakoune implemented
+in Rust.
```

Comment:
DESCRIPTION starts on the same line, lines can go up to 100 characters now (this has been changed recently): https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#general-tips